webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/queries/CmdbQuery.scala

```diff
@@ -947,6 +947,14 @@ object ResultTransformation {
   }
 }
 
+object Query{
```

Comment:
Please add a comment "necessary for 6.2 binary compat, need to be removed in 7.0 to avoid ambiguities in constructor"